### PR TITLE
Cheat Sheet: Fixes #1690

### LINKED ISSUE #1690: Currency Cheat Sheet: Add currency symbols to content
Because the Cheat Sheet Goodie allows ['symbols'](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/CheatSheets.pm#L23) as a trigger word the search term ['currency symbols'](https://duckduckgo.com/?q=currency+symbols) will display the currency cheat sheet, which - unfortunately- doesn't contain any symbol for the listed currencies. This behaviour might be confusing for the user.

-----
IA Page: https://duck.co/ia/view/currency_cheat_sheet

### PR DESCRIPTION
**Pull request by amitdev**
Added currency symbols with Unicode symbols wherever available.

-----
IA Page: https://duck.co/ia/view/currency_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@amitdev Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by moollaza** [comment]
@amitdev thanks for this! It looks like there are merge conflicts though -- can you please resolve them?

**Comment by moollaza** [comment]
It may also be simplest to just save this file as it is, and start with a fresh branch that's up to date with duckduckgo/master

**Comment by amitdev** [comment]
@moollaza I merged the file since it was formatted better.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=currency cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/currency_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/currency.json [hunk 1]
```diff
@@ -64,252 +64,253 @@
...
-                "val": "Euro"
+                "val": "€ (Euro)"
             },
             {
                 "key": "Belgium",
-                "val": "Euro"
+                "val": "€ (Euro)"
             },
             {
                 "key": "France",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/currency.json]: This one isn't quite right -- it looks like this currency is only used for some french islands? France uses the euro

File: share/goodie/cheat_sheets/json/currency.json [hunk 2]
```diff
@@ -40,23 +40,23 @@
             },
             {
                 "key": "Egypt",
-                "val": "Egyptian pound"
+                "val": "£ (Egyptian pound)"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/currency.json]: I feel like `Pound`, `Dollar` etc should be capitalized, but Wiki doesn't seem to care about that. Seeing as these aren't proper sentences I think we should Title Case the `val`'s -- what do you think?

File: share/goodie/cheat_sheets/json/currency.json (hunk 1 above)
Comment (amitdev) [inline share/goodie/cheat_sheets/json/currency.json]: Good catch! I just added the symbols and didn't check original data :wink: Anyway fixed now.

File: share/goodie/cheat_sheets/json/currency.json (hunk 2 above)
Comment (amitdev) [inline share/goodie/cheat_sheets/json/currency.json]: Done

**Comment by amitdev** [comment]
Looks like the source URL was also wrong all this while - fixed it.

File: share/goodie/cheat_sheets/json/currency.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/currency.json]: :+1:

**Comment by moollaza** [comment]
@amitdev this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**